Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -17,10 +17,18 @@ static QString GetSettingsPath()
   return QString::fromStdString(File::GetUserPath(D_CONFIG_IDX)) + QStringLiteral("/UI.ini");
 }
 
-Settings::Settings(QObject* parent) : QSettings(GetSettingsPath(), QSettings::IniFormat, parent)
+Settings::Settings() : QSettings(GetSettingsPath(), QSettings::IniFormat)
 {
 }
 
+Settings& Settings::instance()
+{
+  static Settings* s_settings = nullptr;
```

Comment:
Okay, just to clarify, this is guaranteed to be initialized when the function is called (not during global static initialization), since C++11, right?